Documentation/CreateReactApp.md

```diff
@@ -0,0 +1,45 @@
+This file is probably out of date. Brendan fix pls.
+
+## Table of Contents
+- [Available Scripts](#available-scripts)
+  - [test](#test)
+  - [lint](#lint)
+  - [eject](#eject)
+  - [dev-start](#dev-start)
+  - [Packaging](#packaging)
+
+## Available Scripts
+
+In the project directory, you can run these commands using `npm` or `yarn` although `yarn` is preferred do to the performance benefits:
+
+### `test`
+
+Currently this project has no tests. They will be added at a future date.
+
+Launches the test runner in the interactive watch mode.<br>
+
+### `lint`
+
+Runs the linter
+
+### `eject`
+
+**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+
+If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+
+Instead, it will copy all the configuration files and the transitive dependencies (Webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+
+You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+
+### `dev-start`
+From the `frontend` directory, type `start-dev`. This will launch an instance of the Electron application.
+
+If you make modifications to the source files and wish to see your changes in the Electron application without stopping and starting it, simply refresh the content within the app using `ctrl-r`/`command-r` depending on your platform of choice.
+
+### Packaging
+
+To build install files execute `yarn build` - by default this will build for your current OS.
+Right now cross-platform builds are not supported.
+
+The resultant binaries are placed into the `/dist/` directory.
```

Comment:
Huh. This isn't what CRA normally spits out (and greatly resembles what _was_ in the root's readme directory <checks old version>... yeah, maybe this should be in `DevReadme.md` or something?

Reply:
Let's have a chat later about what should be in here - merging for now